acceso/templates/acceso/caso_index.html

```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+{% load static %}
+{% load staticfiles %}
+{% get_current_language as LANGUAGE_CODE %}
+
+{% block head %}
+    <script src="http://code.jquery.com/jquery-3.3.1.min.js"></script>
```

Comment:
In base.html, we already have: 
<script src="{% static 'acceso/js/jquery.min.js' %}"></script>
You may not need this import.